slack_sdk/oauth/redirect_uri_page_renderer/__init__.py

```diff
@@ -48,7 +50,7 @@ def render_success_page(
 </head>
 <body>
 <h2>Thank you!</h2>
-<p>Redirecting to the Slack App... click <a href="{url}">here</a></p>
+<p>Redirecting to the Slack App... click <a href="{url}">here</a>. If you use the browser version of Slack, click <a href="{browser_url}" target="_blank">this link</a> instead.</p>
```

Comment:
This is the only change affecting users. I would like to know better sentences for this if you have suggestions.